docs/examples/elements/StepExample/Attached.example.vue

```diff
@@ -1,6 +1,25 @@
 <template lang="html">
   <div>
-    <sui-step-group attached="top">
+    <sui-step-group step-number="three" attached="top">
```

Comment:
should I pass number then map to string, or just pass string like this ?

Reply:
I'd say we should support both. We already have a util function and prop type:
https://github.com/Semantic-UI-Vue/Semantic-UI-Vue/blob/992d1ffdbc691eb9a56074820fc121081bbd14b8/src/lib/mixins/SemanticUIVueMixin/utils.js#L7
https://github.com/Semantic-UI-Vue/Semantic-UI-Vue/blob/992d1ffdbc691eb9a56074820fc121081bbd14b8/src/lib/PropTypes.js#L47

Reply:
oh so you already have util func for it, thank you